### ISSUE TITLE
code completion for js?

### ISSUE THREAD

**Issue by nimo23** [angelozerr/typescript.java#228]
Could typescript.java provide also code completion for js-files (if /typings is installed for that module). Actually it only works if js file is renamed to jsx or ts. So code completion also works for jsx (without renaming to tsx), but it does not work for js-files. I cannot rename all other js files but want to have code completion when using es6-module.

**Comment by nimo23** [comment]
no need for this. I can use Tern.java.

**Comment by angelozerr** [comment]
> Could typescript.java provide also code completion for js-files

Yes it should work. If it doesn't work, it's perhaps your project has JSDT nature. If I rememeber you must activate that with TypeScript preferences (from global / window preferences)

**Comment by nimo23** [comment]
code completion only works if I rename my js-file to a ts-file.

I have this settings:
- Preferences->..->Typescript->"Use Salsa as JavaScript Inference" (enabled)

My "ts.eclipse.ide.core.prefs":

```
eclipse.preferences.version=1
nodeJSPath=/usr/local/bin/node
useNodeJSEmbedded=false
typeScriptBuildPath={"src/js/tsconfig.json"\:{}}
```
